### ISSUE TITLE
Parsing error for IPv6mask `::` is not expanded

### ISSUE THREAD

**Issue by LuWa-at-work** [ip-num/ip-num#87]
Hey im not sure if this is inteded or a bug:
```js
// expanded Address -> works fine on IPv6 and IPv6Mask
const a = new IPv6("ffff:ffff:ffff:ffff:0:0:0:0"); // works fine
console.log(a.toString())  // "ffff:ffff:ffff:ffff:0:0:0:0"
const b = new IPv6Mask("ffff:ffff:ffff:ffff:0:0:0:0"); // works fine
console.log(b.toString())  // "ffff:ffff:ffff:ffff:0:0:0:0"

// collapsed Address -> works only for IPv6 
const c = new IPv6("ffff:ffff:ffff:ffff::"); // works fine
console.log(c.toString())  // "ffff:ffff:ffff:ffff:0:0:0:0"
const d = new IPv6Mask("ffff:ffff:ffff:ffff::");  // throws RangeError (see trace below)
console.log(d.toString()) 
```

Stacktrace
```js
Hexadecatet.ts:50 Uncaught (in promise) RangeError: The number NaN cannot be converted to a BigInt because it is not an integer
    at BigInt (<anonymous>)
    at new Hexadecatet (Hexadecatet.ts:50:67)
    at Hexadecatet.fromString (Hexadecatet.ts:21:16)
    at IPNumber.ts:837:32
    at Array.map (<anonymous>)
    at new IPv6Mask (IPNumber.ts:836:47)
    at setup (PrimeVueTest.vue:26:15)
    at callWithErrorHandling (runtime-core.esm-bundler.js:173:22)
    at setupStatefulComponent (runtime-core.esm-bundler.js:7265:29)
    at setupComponent (runtime-core.esm-bundler.js:7220:11)
```

Version: ip-num@1.5.0

**Comment by dadepo** [comment]
Definitely looks like a bug. I'll take a look into this over the weekend. Thanks for reporting.

**Comment by dadepo** [comment]
@LuWa-at-work Should be fixed in v1.5.1. Thanks for reporting.

**Comment by LuWa-at-work** [comment]
Wow that was fast! thanks a lot 😊
